docs/semconv/database.md

```diff
@@ -261,6 +265,36 @@ This metric is [optional](https://github.com/open-telemetry/semantic-conventions
 |--------------------------|-----------------|------------------|----------------------------------------------------------------------------------------------|
 | `db.overflow.user.count` | UpDownCounter   | `{overflowUser}` | Number of times a User thread attempted to acquire a lock when no locks were available.      |
 
+### Metric: `db.cache.read.ratio`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                  | Instrument Type   | Units (UCUM)       | Description                                                                              |
+|-----------------------|-------------------|--------------------|------------------------------------------------------------------------------------------|
+| `db.cache.read.ratio` | UpDownCounter     | `{cacheReadRatio}` | Percentage of page reads for this buffer pool that were satisfied by a cached page image |
+
+
+## Metric: `db.cache.write.ratio`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                   | Instrument Type   | Units (UCUM)        | Description                                                                               |
+|------------------------|-------------------|---------------------|-------------------------------------------------------------------------------------------|
+| `db.cache.write.ratio` | UpDownCounter     | `{cacheWriteRatio}` | Percentage of page Writes for this buffer pool that were satisfied by a cached page image |
+
+### Metric: `db.lock.waits`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name              | Instrument Type | Units (UCUM)  | Description                         |
+|-------------------|-----------------|---------------|-------------------------------------|
+| `db.lock.waits`   | UpDownCounter   | `{lockWaits}` | Number of threads waiting for lock. |
+
+
+### Metric: `db.lru.writes`
```

Comment:
Does the lru means "Least Recently Used" here?

In the description section of the metrics, could you please add the full name of LRU? Alternatively, at the end of the metrics, you can add a note briefly explaining the meaning of LRU. This will make it clearer.